assets/css/translation.css

```diff
@@ -1,23 +1,48 @@
-html[dir='ltr'] .languageName {
-    padding-left: .5em;
+html[dir='ltr'] .langSelectGroup > *:not(*:last-child) {
+    font-size: 10px;
+    margin-right: 3px;
 }
 
-html[dir='rtl'] .languageName {
-    padding-right: .5em;
+html[dir='rtl'] .langSelectGroup > *:not(*:first-child) {
```

Comment:
Can we combine this rule with the above? I'm not seeing how they're different. Should the above one perhaps be `margin-right` or somethiing?

Reply:
first-child and last child. How to combine these i can factor font-size.

Reply:
`html[dir='ltr'] .langSelectGroup > *:not(*:last-child), html[dir='rtl'] .langSelectGroup > *:not(*:first-child) {`

?

Reply:
Aah okay okay, :)

Reply:
Was this done?

Reply:
Oops i think its done already but I reverted as my UI messed up. Forgot to add back. :)